cypress/integration/navigateHomeAndContactsPages.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('navigateHomeAndContactsPages', function() {
+
+    const operations = [
+        {
+            // Estonian
+            homePage: 'https://ut2019.voog.com/et',
```

Comment:
You could use the base url https://ut2019.voog.com as a configurable variable and not duplicate it across the tests. In a real-world scenario you most probably will run these tests against different environments and it will help you to easily re-configure your tests against dev/test environments.